classes/ActionScheduler_Admin.php

```diff
@@ -0,0 +1,85 @@
+<?php
+
+/**
+ * Class ActionScheduler_Admin
+ */
+class ActionScheduler_Admin {
+	/** @var ActionScheduler_Admin */
+	private static $instance = NULL;
+
+	/**
+	 * Create an instance.
+	 */
+	public static function factory() {
+		if ( null === self::$instance ) {
+			self::$instance = new ActionScheduler_Admin();
+		}
+
+		return self::$instance;
+	}
+	/**
+	 * Initialize the plugin.
+	 */
+	public static function init() {
+		add_action( 'init', [ self::factory(), 'register_scripts' ] );
+		add_action( 'admin_enqueue_scripts', [ self::factory(), 'enqueue_scripts' ] );
+		add_action( 'admin_menu', [ self::factory(), 'register_pages' ], 12 );
+		add_action( 'rest_api_init', [ self::factory(), 'register_api_endpoints' ] );
+	}
+
+	/**
+	 * Register the dashboard page with WooCommerce Admin.
+	 */
+	public function register_pages() {
+		if ( function_exists( 'wc_admin_register_page' ) ) {
+			wc_admin_register_page(
```

Comment:
just an FYI, I'm looking creating a hook for this in https://github.com/woocommerce/woocommerce-admin/pull/2033